data-access-analysis-tools/index-rsp-warnings.rst

```diff
@@ -12,98 +12,60 @@
 
+RSP/IDF: The Missing Features
+------------------------------
 
+Many more features are on our roadmap for the RSP and its related services (Qserv, Butler, etc) and we have requirements to deliver them by the start of the survey. Some major ones to look forward in the future are:
 
-RSP Aspect Functionality Limits
--------------------------------
-
-Some major usability features will not be available, such as support for user database tables; support for parallelized or batch computation; the ability to sync files between RSP accounts and personal devices; and the ability to manage the sharing of data within private groups.
-
-Users installing code: DP0 delegates will be able to add software that can be installed without privileges into their own homespace (eg. via pip install --user) for their personal use to the extent that their quota allows.
-
-
-Bulk Downloads are not Supported
---------------------------------
-
-Bulk downloads: No, bulk data downloads from the RSP specifically, or the IDF in general, are not supported during DP0. This is because bulk download services are not yet in production, and because the major goals of DP0 are to allow Rubin Observatory to evaluate how the community uses the services, and for the community to become familiar with the RSP environment. Note that DESC has made the DC2 DR6 catalogs publicly available via: https://lsstdesc-portal.nersc.gov/.
-
-
-
-Stuff That We Know Does Not Work
---------------------------------
-
-*(Here is space for a list of known issues and stuff that doesn’t work in , e.g., the portal. Especially the portal. To avoid people thinking “This is too hard to figure out” when really it’s just a feature that’s not done.)*
-
-ADQL functionality that does not yet work: 
-AREA
-BOX
-COORDSYS
-COORD1
-COORD2
-INTERSECTS
-
-The only coordinate system currently supported is 'ICRS'.
+- Notebooks: We are planning on providing a filespace that you can access from your personal device (eg laptop) so you can use your favourite editor/IDE to write code in your local environment and save it in a way that makes it visible in your RSP notebook environment
+- Batch: Opportunities for parallelized/batch/non-interactive computation are not currently available
+- Qserv: User tables are not yet available
+- Qserv: TAP/ADQL Queries using some keywords (AREA, BOX, COORDSYS, COORD1, COORD2 and INTERSECTS) are not yet supported
+- Portal: Many improvements are planned, including the ability to start a query in the portal and access it from your notebook
+- Authentication and Authorisation: You will be able to create and manage groups to allow you to share data with specific RSP users
+- API: Virtual Observatory image services are not yet available
+- API: VOSpace service is not yet available
+- ConsolidatedDB: Access to observatory metadata is not yet available
+- Bulk Download Service: Not offered
```

Comment:
I think it's worth keeping the "Note that DESC ..." sentence; it seems part of an exchange of courtesies with them for providing the data.

Reply:
It just feels wrong to be telling people where to get DESC data on a page about RSP risks. Advertise DESC data availability elsewhere. And it seems to be giving mixed messages with "come use our system, but you can also just get the data from over there onto your own cluster". I would prefer to leave that out from here. It can go in the data products description or somewhere.

Reply:
OK, that's fine.  I wasn't really thinking about this being just the "risks" page.